fido_app/models.py

```diff
@@ -25,6 +25,9 @@ class User(db.Model, UserMixin):
     sign_count = db.Column(db.Integer, default=0)
     rp_id = db.Column(db.String(253), nullable=True)
 
+    # Page interaction info
+    tokens = db.relationship('Token', lazy=True, backref=db.backref('user', lazy=True))
```

Comment:
Is this stale? It feels like this should be something closer to this:
```suggestion
    sessions = db.relationship('Session', lazy=True, backref=db.backref('user', lazy=True))
```
It's totally possible that I'm just misremembering

Reply:
You're right. I named it "tokens" before I added a Session model and forgot to change the name afterwards. Fixed.